sdk/storage/storage-file-datalake/README.md

```diff
@@ -137,15 +139,25 @@ Data Lake storage offers three types of resources:
 - A _file system_ in the storage account used via `DataLakeFileSystemClient`
 - A _path_ in a file system used via `DataLakeDirectoryClient` or `DataLakeFileClient`
 
-|Azure DataLake Gen2 	      | Blob       |
-| --------------------------| ---------- |
-|Filesystem                 | Container  | 
-|Path (File or Directory)   | Blob       |
+| Azure DataLake Gen2      | Blob      |
+| ------------------------ | --------- |
+| Filesystem               | Container |
+| Path (File or Directory) | Blob      |
 
 > Note: This client library only supports storage accounts with hierarchical namespace (HNS) enabled.
 
 ## Examples
 
+- [Import the package](#import-the-package "Import the package")
+- [Create the data lake service client](#create-the-data-lake-service-client "Create the data lake service client")
+- [Create a new file system](#create-a-new-file-system "Create a new file system")
+- [List the file systems](#list-the-file-systems "List the file systems")
+- [Create and delete a directory](#create-and-delete-a-directory "Create and delete a directory")
+- [Create a file](#create-a-file "Create a file")
+- [List paths inside a file system](#list-paths-inside-a-file-system "List paths inside a file system")
+- [Download a file and convert it to a string (Node.js)](#download-a-file-and-convert-it-to-a-string-nodejs "Download a file and convert it to a string (Node.js)")
+- [Download a file and convert it to a string (Browsers)](#download-a-file-and-convert-it-to-a-string-browsers "Download a file and convert it to a string (Browsers)")
```

Comment:
Please remove the tooltips here based on the same reason as in https://github.com/Azure/azure-sdk-for-js/pull/10429/files#r467663126

```suggestion
- [Import the package](#import-the-package)
- [Create the data lake service client](#create-the-data-lake-service-client)
- [Create a new file system](#create-a-new-file-system)
- [List the file systems](#list-the-file-systems)
- [Create and delete a directory](#create-and-delete-a-directory )
- [Create a file](#create-a-file)
- [List paths inside a file system](#list-paths-inside-a-file-system)
- [Download a file and convert it to a string (Node.js)](#download-a-file-and-convert-it-to-a-string-nodejs)
- [Download a file and convert it to a string (Browsers)](#download-a-file-and-convert-it-to-a-string-browsers)
```